modules/serverless-kafka-source-yaml.adoc

```diff
@@ -59,3 +59,18 @@ spec:
 ----
 $ oc apply -f <filename>
 ----
+
+.Verification steps
+
+You can verify that the Kafka event source was created:
```

Comment:
I think that the verification steps are still supposed to be in bullets (unnumbered in this case, since it's only one). And in this case, you'd drop the "You can" part.